# Default configuration for ORB Controller
orb_controller:
  # I2C bus number (1 = /dev/i2c-1)
  i2c_bus: 1
  
  # I2C device address (decimal: 8 = 0x08 hex)
  i2c_addr: 8
  
  # Status publishing rate in Hz
  publish_rate: 10.0